### DISCUSSION TITLE
Compiled with problems:

### DISCUSSION THREAD

**Discussion by Kirusshenkin** [remarkjs/remark#935] [Q&A] [answered]
Terminal gives this error, although everything is fine with the import:
![image](https://user-images.githubusercontent.com/25940763/149583485-5b6c0ee9-a3e5-484f-b2d8-00109876e780.png)

Why does this problem occur and how can I fix this problem?

**Comment by ChristianMurphy** [comment, marked as answer]
Sorry you ran into a spot of trouble @Kirusshenkin.
It looks like `hastToReact` is exported from `property-information` https://github.com/wooorm/property-information/blob/897a42b366dced388eeed571dd07cb06845f6486/index.js#L15
The issue you are seeing is likely related to the build tool you are using and how it is configured.
It's hard to tell exactly where or how to fix it, without knowing what tools you are using, what version they're on, how they're configured, etc.

Maybe you could share a sandbox with a runnable example of the issue? [CodeSandbox](https://codesandbox.io/s/github/remarkjs/.github/tree/main/sandbox-templates/react-markdown-with-create-react-app) and [StackBlitz](https://stackblitz.com/github/remarkjs/.github/tree/main/sandbox-templates/react-markdown-with-create-react-app) are two good options.

**Reply by Kirusshenkin** [reply]
Unfortunately, I don't see any mistakes with the code because my colleges said everything is working and there are no problems.  But on my computer Mac M1 there is an issue. 

I have version react-markdown: ^7.1.0

I thought the reason of mistake is version node and npm. Although, I was wrong

In brief, I got this:

`import Markdown from 'react-markdown';`

`<Markdown className='item__text'>{description}</Markdown>`

**Reply by ChristianMurphy** [reply]
Again, I don't think it's an issue with the code per se, but:

> The issue you are seeing is likely related to the build tool you are using and how it is configured.

What build tools are you using? What versions are they? How are they configured?

> my colleges said everything is working and there are no problems

Have you sat down with your colleagues to compare how their system is setup with how yours is?

**Reply by Kirusshenkin** [reply]
You were right. It was about packages. My project has too many packages that prevented each other from working properly. As a result, I found the error and fixed it
